apis/python/src/tiledbsoma/experimental/_xarray_backend.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) 2024 The Chan Zuckerberg Initiative Foundation
+# Copyright (c) 2024 TileDB, Inc
+#
+# Licensed under the MIT License.
+from typing import Any, Mapping, Optional, Tuple, Union
+
+import numpy as np
+from somacore import options
+from xarray import Variable
```

Comment:
i don't understand how this is passing CI ... IIUC `xarray` is a new dependency? Which isn't in our `setup.py` nor in `somacore`'s? 😕

Reply:
It's an indirect dependency through the SpatialData package.

Reply:
I added it as a direct dependency.